### PR TITLE
chore: Set `target="_blank"` on links rendered in OSCI

### PR DESCRIPTION
**Pull request by msugakov**
## Description

I wanted to make it for a long time.

## Checklist
- [x] Investigated and inspected CI test results

Won't do these:
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

### Here I tell how I validated my change

- [x] `Additional StackRox e2e artifacts` links are left-clickable.
- [x] `Groovy Test Logs` links are left-clickable.
- ~~[ ] `GKE Logs Explorer` links are left-clickable~~ - no, they aren't and I could not make them work.

Not creating tests for this change because it's change in tests tooling.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Review by gavin-stackrox — APPROVED** [review]
\o/

**Comment by msugakov** [comment]
Note for self: try https://stackoverflow.com/a/59806930/484050

**Comment by msugakov** [comment]
@gavin-stackrox I could not make the link in `GKE Logs Explorer` left-clickable.
There's something set up in `prow.ci.openshift.org` page that prevents it. I could have gone further to figure it out but I'd give up.

Links in `Groovy Test Logs` were already set up properly so there's no user-visible change.

The net result is that `Additional StackRox e2e artifacts` become properly left-clickable.

Since I got your approval, I'll wait for CI, will check things still work and merge it unless you have more comments in the meantime.

**Pull request merged**
